tests/unit/set_tags/test_set_bucket_tags_handler.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from unittest.mock import MagicMock, patch
+
+from botocore.stub import Stubber
+
+from set_tags import set_bucket_tags
+from set_tags import utils
+
+
+class TestSetBucketTagsHandler(unittest.TestCase):
+
+  def test_happy_path(self):
+    s3 = utils.get_s3_client()
+    with Stubber(s3) as stubber, \
+      patch('set_tags.set_bucket_tags.get_bucket_name') as name_mock, \
+      patch('set_tags.set_bucket_tags.get_bucket_tags') as get_mock, \
+      patch('set_tags.utils.get_principal_id') as arn_mock, \
+      patch('set_tags.utils.get_synapse_user_profile') as profile_mock, \
+      patch('set_tags.utils.get_synapse_tags') as tags_mock:
+        name_mock.return_value = 'some-improbable-bucket-name'
+        tags_mock.return_value = [{ 'Key': 'OwnerEmail', 'Value': 'janedoe@synapse.org' }]
+        stubber.add_response(
+          method='put_bucket_tagging',
+          service_response={
+            'ResponseMetadata': {
+              'RequestId': '12345',
+              'HostId': 'etc',
+              'HTTPStatusCode': 204,
+              'HTTPHeaders': {}
+            }})
+        utils.get_s3_client = MagicMock(return_value=s3)
+        result = set_bucket_tags.create_or_update({},{})
```

Comment:
Shouldn't there be an assertion that 'put_bucket_tagging' passed the expected tags?

Reply:
added this check